### DISCUSSION TITLE
Welcome to neogen Discussions!

### DISCUSSION THREAD

**Discussion by danymat** [danymat/neogen#136] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bryant-the-coder** [comment]
thank you for having the discussions page!

**Reply by danymat** [reply]
I thought it was a great addition ! :)

**Reply by bryant-the-coder** [reply]
indeed it is!!
